etc/ci/buildbot_steps.yml

```diff
@@ -27,6 +27,7 @@ mac-rel-wpt4:
   - ./mach build --release
   - ./mach test-wpt --release --processes 4 --total-chunks 4 --this-chunk 4 --log-raw test-wpt.log --log-errorsummary wpt-errorsummary.log --always-succeed
   - ./mach filter-intermittents wpt-errorsummary.log --log-intermittents intermittents.log --log-filteredsummary filtered-wpt-errorsummary.log --use-tracker
+  - ./mach test-wpt --release --binary-arg=multiprocess --processes=8 --pref dom.servoparser.async_html_tokenizer.enabled --log-raw test-wpt.log --log-errorsummary wpt-errorsummary.log domparsing html/syntax html/dom/documents html/dom/dynamic-markup-insertion
```

Comment:
Also, we need to be careful about what we do with the logs, or they will overwrite the ones from earlier commands.